pkg/cmd/upgrade.go

```diff
@@ -29,6 +29,7 @@ type upgradeOpts struct {
 	globalOptions
 	ForceUpgrade              bool `longflag:"force" shortflag:"f"`
 	UpgradeMachineDeployments bool `longflag:"upgrade-machine-deployments"`
+	PruneImages               bool `longflag:"prune-images"`
```

Comment:
we don't need to update that deprecated and hidden subcommand. In fact I'm surprised it still exists and we forgot to remove it and install subcommand :man_shrugging: :joy:

Reply:
OK, so together with @mohamed-rafraf we've decided to go ahead and remove those commands in the follow up PR, so I'm /lgtm /approve this one (to avoid additional retesting).